### PR TITLE
Add user model after creation of user table

### PR DESCRIPTION
**Pull request by msm-faraj**

### CODE REVIEW

File: src/models/user.js [hunk 1]
```diff
@@ -0,0 +1,74 @@
+const { Sequelize, Datatypes, Model } = require("sequelize");
+
+module.exports = (sequelize, DataTypes) => {
+  class User extends Model {
```
Comment (pouyapourbaba) [inline src/models/user.js:4]: Can you check the Sequelize docs for other ways of creating a model?

Comment (msm-faraj) [reply src/models/user.js]: done

File: src/models/user.js [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+      },
+      deletedAt: {
+        type: DataTypes.DATE,
+        allowNull: true,
+      },
+    },
+    {
+      paranoid: true, // Enables soft deletes
+      tableName: "Users", // Optional: specify table name explicitly
+    }
```
Comment (pouyapourbaba) [inline src/models/user.js:66]:
```suggestion
```

File: src/models/user.js [hunk 3]
```diff
@@ -0,0 +1,70 @@
+const { DataTypes } = require("sequelize");
```
Comment (pouyapourbaba) [inline src/models/user.js:1]:
```suggestion
```

**Review by pouyapourbaba — APPROVED** [review]

**Pull request merged**